gosearch.go

```
@@ -63,64 +62,290 @@ var tlsConfig = &tls.Config{
+	wg.Add(1)
+	go SearchDomains(username, domains, &wg)
+	wg.Wait()
+
+	fmt.Println()
+	fmt.Println()
+
+	// Calculate and display elapsed time
+	elapsed := time.Since(start)
+
+	table := tablewriter.NewTable(os.Stdout, tablewriter.WithRenderer(renderer.NewBlueprint(tw.Rendition{Borders: tw.BorderNone})))
+	table.Append(Bold("Number of profiles found"), Red(count.Load()))
```

golangci-lint (1.64.8)

Error return value of `table.Append` is not checked

(errcheck)